helm/kyverno-policies-connectivity/templates/PodController.yaml

```diff
@@ -0,0 +1,135 @@
+# THIS FILE IS GENERATED WITH 'make generate' - DO NOT EDIT MANUALLY
+{{- if .Values.proxy.enabled }}
+{{- $proxy := deepCopy .Values.cluster.proxy | mustMerge .Values.proxy }}
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: inject-proxy-env
+spec:
+  rules:
+    - name: inject-proxy-env-to-containers
+      exclude:
+        any:
+        - resources:
+            kinds:
+            - Deployment
+            - StatefulSet
+            - DaemonSet
```

Comment:
what about jobs/cronjobs? I think they were covered in the `Pod.yaml`

Reply:
Yeah, good point! I think adding only `Job` would be enough, right? Because `Cronjobs` ultimately just create `Jobs`.